pootle/apps/pootle_app/management/commands/flush_cache.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) Pootle contributors.
+#
+# This file is a part of the Pootle project. It is distributed under the GPL3
+# or later license. See the LICENSE file for a copy of the license and the
+# AUTHORS file for copyright and authorship information.
+
+import os
+
+# This must be run before importing Django.
+os.environ['DJANGO_SETTINGS_MODULE'] = 'pootle.settings'
+
+from django.core.management.base import BaseCommand
+
+from django_redis import get_redis_connection
+
+from pootle.core.models import Revision
+from pootle_store.models import Unit
+
+
+class Command(BaseCommand):
+    help = """Flush cache."""
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--default',
+            action='store_true',
+            dest='flush_default',
+            default=False,
+            help='Flush default cache',
```

Comment:
Can we not call this default? Thought its named that.  Lets name it after what it contains.